MM-31717: Remove HTTP clustering (#4545)
mattermost/docs: source/deployment/cluster.rst

8. When the server is running, start the other servers.
9. Restart NGINX.

All Cluster Nodes Must Use a Single Protocol
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> From Mattermost Server v5.35, `gossip clustering can no longer be disabled <https://docs.mattermost.com/administration/config-settings.html#use-gossip>`__. All cluster traffic uses the gossip protocol.

Reviewer: ```suggestion
All cluster traffic uses the gossip protocol. From Mattermost Server v5.35, `gossip clustering can no longer be disabled <https://docs.mattermost.com/administration/config-settings.html#use-gossip>`__.
```

> All cluster traffic uses the gossip protocol. From Mattermost Server v5.35, `gossip clustering can no longer be disabled <https://docs.mattermost.com/administration/config-settings.html#use-gossip>`__.

Author: ```suggestion
All cluster traffic uses the gossip protocol. From Mattermost Server v5.36, `gossip clustering can no longer be disabled <https://docs.mattermost.com/administration/config-settings.html#use-gossip>`__.
```

> When upgrading a High Availability cluster, you can't upgrade other nodes in the cluster when one node isn't using the gossip protocol. You must use gossip to complete a High Availability upgrade, or you can shut down all nodes and bring them all up following an upgrade.

Reviewer: ```suggestion
When upgrading a High Availability cluster, you can't upgrade other nodes in the cluster when one node isn't using the gossip protocol. You must use gossip to complete a High Availability upgrade. Alternatively you can shut down all nodes and bring them all up individually following an upgrade.
```
